oraichain/assetlist.json

```diff
@@ -720,8 +718,100 @@
             "background_color_hex": "#0088CC"
           }
         }
+      ]
+    },
+    {
+      "description": "The PEPE native token of Oraichain",
+      "denom_units": [
+        {
+          "denom": "factory/orai1wuvhex9xqs3r539mvc6mtm7n20fcj3qr2m0y9khx6n5vtlngfzes3k0rq9/extPEPE",
+          "exponent": 0
+        },
+        {
+          "denom": "pepe",
+          "exponent": 6
+        }
+      ],
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "ethereum",
+            "base_denom": "0x6982508145454Ce325dDbE47a25d4ec3d2311933"
+          },
+          "provider": "Oraichain Labs OBridge"
+        }
+      ],
+      "base": "factory/orai1wuvhex9xqs3r539mvc6mtm7n20fcj3qr2m0y9khx6n5vtlngfzes3k0rq9/extPEPE",
+      "name": "Pepe",
+      "display": "pepe",
+      "symbol": "PEPE",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/pepe.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/pepe.svg"
+      },
+      "images": [
+        {
+          "image_sync": {
+            "chain_name": "ethereum",
+            "base_denom": "0x6982508145454Ce325dDbE47a25d4ec3d2311933"
+          },
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/pepe.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/pepe.svg",
+          "theme": {
+            "primary_color_hex": "#cc3233"
+          }
+        }
+      ],
+      "keywords": ["meme"]
+    },
+    {
+      "description": "The Hamster Kombat native token of Oraichain",
+      "denom_units": [
+        {
+          "denom": "factory/orai1wuvhex9xqs3r539mvc6mtm7n20fcj3qr2m0y9khx6n5vtlngfzes3k0rq9/HMSTR",
+          "exponent": 0,
+          "aliases": ["nanoHmstr"]
+        },
+        {
+          "denom": "HMSTR",
+          "exponent": 9
+        }
+      ],
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "ton",
+            "base_denom": "nanoHmstr"
+          },
+          "provider": "Oraichain Labs Ton Bridge"
+        }
+      ],
+      "base": "factory/orai1wuvhex9xqs3r539mvc6mtm7n20fcj3qr2m0y9khx6n5vtlngfzes3k0rq9/HMSTR",
+      "name": "HMSTR",
+      "display": "hmstr",
```

Comment:
this denom (case sensitive) isn't defined in denom_units. Either update this to "HMSTR" to make the defined unit, or change the exponent:9 denom unit to "hmstr" so it matches this.